transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionSubsystemRootResourceDefinition.java

```diff
@@ -82,6 +83,11 @@
     public static final RuntimeCapability<Void> TRANSACTION_SYNCHRONIZATION_REGISTRY_CAPABILITY =
             RuntimeCapability.Builder.of("org.wildfly.transactions.transaction-synchronization-registry", TransactionSynchronizationRegistry.class)
                     .build();
+
+    public static final RuntimeCapability<Void> REMOTE_TRANSACTION_SERVICE_CAPABILITY =
+            RuntimeCapability.Builder.of("org.wildfly.transactions.remote-transaction-service", RemotingTransactionService.class)
+                    .build();
```

Comment:
@mmusgrov Any concerns about this?  To me RemotingTransactionService seems like a reasonable API to expose via a capability, and in any case it's already being used in other subsystems for MSC service wiring.  This addition of a capability just formalizes the contract a bit.